### ISSUE TITLE
GUI - problem prompt

### ISSUE THREAD

**Issue by Kamila-Bober** [qMRLab/qMRLab#485]
I'm currently working on the B0_dem and B0_mem (multiple echo method), and I would like to create error prompt for the user. In case of bad data entry. I was wondering how I should go about it. What files should I focus on?

**Comment by agahkarakuzu** [comment]
Hi @Kamila-Bober there are already some data dimensionality checks in place, so I assume that you'd like to prompt messages for handling a different case? If you can give us some details, I can help you. 

Is B0_dem you are working on different than the one that has been implemented (`b0_dem`) in qMRLab?

**Comment by mathieuboudreau** [comment]
@agahkarakuzu yes her method is different - I'm meeting with her tomorrow morning to show her how to make a branch and push it so we can see her new model's code =)
